BUILDER_DEV.md

```diff
@@ -1,225 +1,106 @@
-
 ## Overview
 
 This document captures the steps to start and run a Builder environment for development. The builder environment includes the builder services, as well as the depot web site.
 
 There are several ways of creating a Builder dev environment - but supporting all operating systems and environments has proven to be untenable. This document includes one officially supported way of creating a Builder dev environment, and links to unsupported ways of creating the dev environment that you may use at your own risk.
 
-Eventually, we will want to harness the power of the Habitat studio to automate a builder dev environment, but this serves as a stop gap for now.
-
-## Officially Supported Dev Environment
-
-This Dev Environment involves spinning up a virtual machine (most of the core contributors use VMWare, but Virtual Box should work as well) running Ubuntu 17.10 desktop OR Ubuntu 17.10 Server.
-
-### Pre-Reqs
-
-* Ubuntu 17.10 Desktop or Ubuntu 17.10 Server Virtual Machine
-* Access to the "core" origin in [Production Builder](https://bldr.habitat.sh). Ask a core maintainer (or in the [Habitat Slack](http://slack.habitat.sh/)) if you do not have access.
-* Access to the Habitat 1Password account (NOTE - we are working on a solution to make this not required, but it is necessary for the time being)
-* All of the following steps should be run within your Virtual Machine
-* A [Github Personal Access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with all repo and all user permissions.
-* The [Habitat Builder App](https://github.com/apps/habitat-builder) installed on your Github account.
-* Remove the "secure_path" section from your sudoers file (this is required for the make scripts to run correctly). The following command will comment it out:
-```
-$ sudo sed -i.bak '/secure_path/s/^/#/' /etc/sudoers
-```
-* Some of the sample commands below use the 'httpie' tool. Install it if not present on your system (https://github.com/jkbrzt/httpie).
-
-
-### Builder Setup
-* Update your system
-```
-$ sudo apt-get update
-```
-
-* NOTE - if you are running Ubuntu 17.10 Server, the above command may fail with "E: The repository 'cdrom://Ubuntu-Server 17.10 _Artful Aardvark_ - Release amd64 (20171017.1) artful Release' does not have a Release file."  To fix this, run:
-
-```
-$ sudo sed -i.bak '/deb cdrom/s/^/#/g' /etc/apt/sources.list
-```
+## Pre-Requisites
 
-* Install git
-```
-$ sudo apt-get install git
-```
+### Ports required
+1. 9636 - Intra-supervisor communication
+1. 80 - Web app
+1. 9631 - supervisor api port
+1. 5432 - posgres port
 
-* Clone the [Habitat repo](https://github.com/habitat-sh/habitat)
-```
-$ git clone https://github.com/habitat-sh/habitat.git
-```
+If you are using Linux environment you can run `support/linux/provision.sh` to configure your host
```

Comment:
I think we need something here about having the repo which contains `provision.sh`, either via `git clone` or downloading the source directly.